static/app/components/assistant/getGuidesContent.tsx

```diff
@@ -241,14 +241,14 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
           title: t('Transactions'),
           target: 'trace_view_guide_row',
           description: t(
-            `Get an overview of every transaction. You can quickly see the all the transactions in a trace alongside the project, transaction duration, and any related errors.`
+            `Get an overview of every transaction. You can quickly see all the transactions in a trace alongside the project, transaction duration, and any related errors.`
           ),
         },
         {
           title: t('Transactions Details'),
           target: 'trace_view_guide_row_details',
           description: t(
-            `Click on any transaction to get see more details and even go to its transaction details.`
+            `Afraid you're missing something? Click on any transaction fo' mo' details. FOMO no mo'!`
```

Comment:
I'm not a huge fan of this. FOMO is definitely slang that not everyone will get.

Reply:
Yeah I can see that. Original was hard to salvage ([details? details?](https://www.youtube.com/watch?v=vXlST7il84I)), so I started from scratch. Prefer straightforward "Click on any transaction to see more details"?

Reply:
Done in 754bf246ba0312172d540bc4787ef24cb3a626ea. 👍

Reply:
(Ftr, I kinda figure if anyone is like "what is FOMO?" then google.com/search?q=fomo and it's like laaaayers of humour but let's at least fix the straight up typos if we can't all agree on the brilliance. 🤩)